Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -953,7 +953,7 @@ public final class PackageBuilder {
             {
                 self.observabilityScope
                     .emit(
-                        warning: "'\(potentialModule.name)' was identified as an executable target given the presence of a 'main.swift' file. Starting with tools version \(ToolsVersion.v5_4) executable targets should be declared as 'executableTarget()'"
+                        warning: "'\(potentialModule.name)' was identified as an executable target given the presence of a 'main' file. Starting with tools version \(ToolsVersion.v5_4) executable targets should be declared as 'executableTarget()'"
```

Comment:
How about "given the presence of an entry point file"? Makes it a bit more general.

Reply:
`due to presence of ...` to make it slightly shorter?